demo/custom-drawer-toggle-attribute.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
+<!--
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+  <head>
+    <title>paper-drawer-panel demo</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    
+    <link rel="import" href="../../paper-styles/demo-pages.html">
+    <link rel="import" href="../../paper-button/paper-button.html">
+    <link rel="import" href="../paper-drawer-panel.html">
+
+    <style is="custom-style">
+
+      #paperDrawerPanel [main] {
+        background-color: var(--google-grey-100);
+      }
+
+      #paperDrawerPanel [drawer] {
+        border-right: 1px solid var(--google-grey-300);
+      }
+
+      #paperDrawerPanel[right-drawer] [drawer] {
+        border-left: 1px solid var(--google-grey-300);
+      }
+
+      paper-button {
+        color: white;
+        margin: 10px;
+        background-color: var(--google-blue-700);
+        white-space: nowrap;
+      }
+
+      paper-drawer-panel:not([narrow]) [custom-toggle] {
+        display: none;
+      }
+
+    </style>
+  </head>
+
+  <body unresolved>
+
+    <paper-drawer-panel id="paperDrawerPanel" drawer-toggle-attribute="custom-toggle">
+      <div drawer>
+      </div>
+      <div main>
+        <div>
+          <p>Custom drawer-toggle-attribute</p>
+          <paper-button custom-toggle raised>toggle drawer</paper-button>
+        </div>
+      </div>
+    </paper-drawer-panel>
+
+    <script>
+      (function(global) {
+
+        'use strict';
+
+        var DRAWER_ATTR = 'right-drawer';
+
+        var pdp = document.getElementById('paperDrawerPanel');
+
+        global.flipDrawer = function() {
```

Comment:
Do we need this for this demo?  I don't see the button that triggers `flipDrawer`